uptrain/operators/embs.py

```diff
@@ -90,11 +94,11 @@ class Distribution(TableOp):
     _agg_func: t.Callable | None = None
 
     @root_validator(pre=True)
-    def check_cols(cls, values):
+    def _check_cols(cls, values):
         """
         Validator to check the validity of input and output column lists.
 
-        Args:
+        Attributes:
```

Comment:
Nit - functions have args. classes have attributes

Reply:
Yeah my bad, fixed it